CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 
+- Add missing translation within search autocomplete menu
```

Comment:
I would put it as a fix since it should have been translated. It's not really a new feature :stuck_out_tongue: